### PR TITLE
Feature/adding job

### LINKED ISSUE #18: Create a working "one job" example
Please use your latest work (example 2) to add a job thanks to copy/paste from a working Jenkins instance.

### PR DESCRIPTION
**Pull request by ash-sxn**
adding the job named `hello`
which creates a file named hello.txt with  `Hello World!` inside it with `echo Hello World! > hello.txt` in the first step
then view the file with `cat hello.txt`
Here's the Script used :
```
pipeline {
  agent any
  stages {

    stage('First') {
      steps {
        sh 'echo Hello World! > hello.txt'
      }
    }
    
    stage('Second') {
      steps {
        sh 'cat hello.txt'
      }
    }

  }
}
```
Please tell if I should use a different job 
- This resolves #18 
- Pinging mentors @jmMeessen @gounthar @berviantoleo

### CODE REVIEW

**Review by gounthar — CHANGES_REQUESTED** [review]
Thanks.
We don't need the files in the `builds` directory.

**Comment by ash-sxn** [comment]
Sorry for that
I've deleted the build directory

**Review by gounthar — APPROVED** [review]

File: experimental_docker_compose_files/02_custom_docker_file_connecting_agent_and_controller/dockerfiles/jobs/hello/config.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <script>pipeline {
+  agent any
+  stages {
+
+    stage(&apos;First&apos;) {
+      steps {
+        sh &apos;echo Hello World! &gt; hello.txt&apos;
+      }
+    }
+
```
[the remark this answers is not in the document]
Comment (ash-sxn) [reply experimental_docker_compose_files/02_custom_docker_file_connecting_agent_and_controller/dockerfiles/jobs/hello/config.xml:26]: how about adding sleep before the command?
like for 3-4 seconds

File: experimental_docker_compose_files/02_custom_docker_file_connecting_agent_and_controller/dockerfiles/jobs/hello/config.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  <actions>
+    <org.jenkinsci.plugins.pipeline.modeldefinition.actions.DeclarativeJobAction plugin="pipeline-model-definition@2.2141.v5402e818a_779"/>
+    <org.jenkinsci.plugins.pipeline.modeldefinition.actions.DeclarativeJobPropertyTrackerAction plugin="pipeline-model-definition@2.2141.v5402e818a_779">
+      <jobProperties/>
+      <triggers/>
+      <parameters/>
+      <options/>
+    </org.jenkinsci.plugins.pipeline.modeldefinition.actions.DeclarativeJobPropertyTrackerAction>
+  </actions>
+  <description>hello &#xd;
```
Comment (jmMeessen) [inline experimental_docker_compose_files/02_custom_docker_file_connecting_agent_and_controller/dockerfiles/jobs/hello/config.xml:12]: For the intended audience, the title of the job is not indicative. How about changing it to "(simple) demo job"

File: experimental_docker_compose_files/02_custom_docker_file_connecting_agent_and_controller/dockerfiles/jobs/hello/config.xml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+this is for the first sample job which first creates a file hello.txt with &apos;hello word!&apos; inside it and then cat this file</description>
```
Comment (jmMeessen) [inline experimental_docker_compose_files/02_custom_docker_file_connecting_agent_and_controller/dockerfiles/jobs/hello/config.xml:13]: Suggestion to improve the wording:

```suggestion
A simple pipeline based job. It creates a hello.txt  file with &apos;hello word!&apos; inside it and then displays the file.</description>
```

**Comment by ash-sxn** [comment]
sorry for the delay, slipped out of the mind

**Review by jmMeessen — APPROVED** [review]
LGTM

**Pull request merged**